README.md

```diff
@@ -573,6 +573,79 @@ Don't worry about `adbkey` or `adbkey.pub` under `/.android`, not required.
+* Create a project in [Firebase](https://console.firebase.google.com/):
+    * Choose the recently created Google Cloud Platform project to add Firebase services to it.
+    * Confirm Firebase billing plan.
+* Go to **IAM & Admin** -> **Service Accounts** in [Google Cloud Platform](https://console.cloud.google.com/iam-admin/serviceaccounts):
+    * Edit the Firebase Admin SDK Service Agent account.
+    * **Keys** -> **ADD KEY** -> **Create new key** -> **Key type**: **JSON** -> **CREATE**.
+    * Download and save the created private key to your computer.
+* Go to **IAM & Admin** -> **IAM** in [Google Cloud Platform](https://console.cloud.google.com/iam-admin/iam):
+    * Edit the Firebase Admin SDK Service Agent account.
+    * **ADD ANOTHER ROLE** -> **Role**: **Project** -> **Editor** -> **SAVE**.
+* Go to [API Library](https://console.developers.google.com/apis/library) -> search for **Cloud Testing API** and **Cloud Tool Results API** -> enable them.
+
+* Now let's pull and run the Docker image with Google Cloud SDK
+
+```bash
+# pull the image
+docker pull thyrlian/android-sdk-firebase-test-lab
+# run the container passing the service account Json (at least the first time) as external volume
+docker run -d -p 2222:22 -v $(pwd)/sdk:/opt/android-sdk -v /path/on/host/auth.json:/auth.json thyrlian/android-sdk-firebase-test-lab
+
+# check the running process and get a interactive shell
+docker exec -it `docker ps -aqf "ancestor=thyrlian/android-sdk-firebase-test-lab"` /bin/bash
+
+# authenticate and create any config that you want
+gcloud auth activate-service-account -q --key-file myservicefile.json
+
+# list available devices to run your tests
+gcloud firebase test android models list
+
+# you should get a matrix output similar to this one but bigger
+┌───────────────────┬────────────────────┬──────────────────────────────────────┬──────────┬─────────────┬─────────────────────────┬───────────────┐
+│      MODEL_ID     │        MAKE        │              MODEL_NAME              │   FORM   │  RESOLUTION │      OS_VERSION_IDS     │      TAGS     │
+├───────────────────┼────────────────────┼──────────────────────────────────────┼──────────┼─────────────┼─────────────────────────┼───────────────┤
+│ Nexus9            │ HTC                │ Nexus 9                              │ VIRTUAL  │ 2048 x 1536 │ 21,22,23,24,25          │               │
+│ NexusLowRes       │ Generic            │ Low-resolution MDPI phone            │ VIRTUAL  │  640 x 360  │ 23,24,25,26,27,28,29,30 │ beta=30       │
+│ OnePlus3T         │ OnePlus            │ OnePlus 3T                           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
+└───────────────────┴────────────────────┴──────────────────────────────────────┴──────────┴─────────────┴─────────────────────────┴───────────────┘
+
+# compile your apks both app and testing
```

Comment:
both -> for both
testing -> instrumented tests  (because testing could be also local JVM unit tests, it may cause confusion).